constitution.tex

```diff
@@ -794,6 +794,9 @@
 	\item The candidates will be notified of their nomination.
 		Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 		A list of all nominees who have accepted their nominations will be posted shortly thereafter.
+	\item The date and time of the candidate(s)'s speeches will be announced by a member of the Executive Board at least five (5) days prior to the event. 
+		All Active Members are expected to attend candidate speeches as specified in \ref{Expectations of House Members}.
```

Comment:
this reads as though candidate speeches are defined in expectations

Reply:
Should I change `as specified in` to `in accordance of`?

Reply:
`in accordance with` would be my choice